pot_roast.txt

```diff
@@ -0,0 +1,14 @@
+sized for 75-100 ppl
+
+30 pounds beef chuck roast in 9 chunks
```

Comment:
So my usual target is to reduce meat consumption to 75-100g per person per meal. So to scale this at 100 persons we need to cut this in half.

Reply:
this recipe is just the "a chunk of meat" part of a meal - you'd probably serve it with sides of some sort in order to have a smaller portion of meat per person.